doc/classes/Particles.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Particles" inherits="GeometryInstance" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		3D particle emitter.
 	</brief_description>
 	<description>
+		3D particle node used to create a variety of particle systems and effects. Particles features an emitter that generates some number of particles at a given rate.
```

Comment:
3D particle node used to create a variety of particle systems and effects. [code]Particles[/code] features an emitter that generates some number of particles at a given rate.